backend/tests/userTests.test.js

```diff
@@ -0,0 +1,89 @@
+// M6 Q3
+//import {jest} from '@jest/globals';
+//import axios from 'axios';
+var axios = require('axios');
+const { ObjectID } = require('bson');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
+
+jest.mock('axios');
+
+//UserStore.js tests
+
+test('Get user tokens', () => {
+    const req = '1, 2, 3';
+    const res = [{'userID': 1, 'token': 'a'}, {'userID': 2, 'token': 'b'}, {'userID': 3, 'token': 'c'}];
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserStoreURL + '/getUserTokens?userids=' + req).then(response => expect(response).toEqual(res))
+});
+
+
+test('Store user token', () => {
+    const req = {'userID': 1, 'token': 'a'};
+    const res = {'result': 'Sucessfully stored user token'};
+
+    axios.post = jest.fn().mockResolvedValue(res);
+    return axios.post(UserStoreURL + '/storeUserToken', {
+        method: 'POST',
```

Comment:
Codacy has a fix for the issue: [Replace `··method:·'POST'` with `method:·"POST"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
      method: "POST",
```